### DISCUSSION TITLE
Why not 869,075 Mhz in EU

### DISCUSSION THREAD

**Discussion by JaraLowell** [meshtastic/firmware#3954] [Q&A] [answered]
While i would say 869,525 Mhz might work in some country's here in Holland we have a ton of other things right on that frequency as well STG Series WMBus some farmers seem to use for sensors and in some solar panels (some time complete parks) in my neighborhood a small 3 sec gave me this on the sdr waterfall alone

![Screenshot of SDRPP waterfall on 869525 Mhz 125bandwith.](https://i.gyazo.com/a3b45a271f6526ffee274fc6547ddbe6.png)

While if i look at local laws for Holland 869,075 with 250 Bandwidth would also still be legal and not be on that one frequency all are on, or am i missing something here ?

**Comment by ianatkin80** [comment, marked as answer]
I believe the regulations are the same across the EU, you can only use 869.525 if you want 500mW and 10% duty cycle. Other frequencies like the one you mentioned have a 25mW limit and 1% duty cycle so it wouldn't work for a city wide mesh.
